### DISCUSSION TITLE
Allow selecting link type

### DISCUSSION THREAD

**Discussion by FL33TW00D** [Textualize/rich#2194] [General] [answered]
Hi all,
I am looking to use Rich in a Jupyter Notebook. This PR sets the target to _blank for links in Jupyter mode.
https://github.com/Textualize/rich/pull/1945

However, I am using a link within a panel, as follows:
```python
    api_prompt = Panel(
        (
            f"You can find your API key :key: in your browser"
            f" [blue underline][link={web_url}/authorize]here[/link][/blue underline] "
        ),
        box=box.DOUBLE_EDGE,
    )
```

As I am not using `print` from `rich.jupyter`, the link does not open in a new tab.

Would it be possible to optionally configure the target of the link in the prompt itself? Or is there a way with the current options I have missed?

Many thanks

**Comment by willmcgugan** [comment]
It works for me. Rich will always generate the `target=blank` Are you familiar with browser dev tools?

**Reply by FL33TW00D** [reply]
Hi Will,
<img width="1137" alt="no_blank" src="https://user-images.githubusercontent.com/45471420/164404088-edbe45f5-6679-41c2-86dc-84df3a8c708d.png">
It seems that the blank is not being generated for me. This line is how the message is produced 

```python
display = Console()
api_prompt = Panel(
 (
      f"You can find your API key :key: in your browser"
      f" [blue underline][link={web_url}/authorize]here[/link][/blue underline] "
 ), box=box.DOUBLE_EDGE 
)
display.print(api_prompt, style="bold yellow")
```

My dependencies:
```toml
[tool.poetry.dependencies]
python = "^3.8"
rich = "^11.0.0"
ipywidgets = "^7.6.5"
```

Let me know if you want me to create a minimal reproducible example.

Many Thanks

**Comment by willmcgugan** [comment, marked as answer]
The `target="_blank"` was introduced in 12.0.0. I suspect you just need to upgrade Rich in your kernel.

**Reply by FL33TW00D** [reply]
My apologies - works perfectly after upgrading.
